src/main/resources/common-services/CDAP/4.0.0/configuration/cdap-env.xml

```diff
@@ -45,6 +45,19 @@
   </property>
 
   <property>
+    <name>yum_gpgcheck_enabled</name>
+    <value>true</value>
+    <description>
+      Enable gpg check
```

Comment:
GPG... and maybe add more context like "Enable Yum GPG signature check on repository packages"